portal-frontend/src/app/features/applications/view-submission/view-application-submission.component.html

```diff
@@ -126,5 +114,47 @@ <h2>Applicant Submission</h2>
         </div>
       </mat-tab>
     </mat-tab-group>
+    <div *ngIf="application && application.canEdit" class="continue-button-container">
```

Comment:
I feel like this is no longer just a continue button container. Maybe change to just `button-container`?

Reply:
same as above